session/handler.go

```diff
@@ -440,6 +440,29 @@ func (h *Handler) adminGetSession(w http.ResponseWriter, r *http.Request, ps htt
 	h.r.Writer().Write(w, r, sess)
 }
 
+// swagger:route DELETE /admin/sessions/{id} v0alpha2 adminRevokeSession
+//
+// # Invalidate a Session from an Administrative context
+//
+// Calling this endpoint invalidates the specified session. The current session cannot be revoked.
+// Session data is not deleted.
+//
+// This endpoint is useful for:
+//
+// - To forcefully logout the current user from another device or session
+//
+//	Schemes: http, https
+//
+//	Responses:
+//	  204: emptyResponse
+//	  400: jsonError
+//	  401: jsonError
+//	  500: jsonError
+func (h *Handler) adminRevokeSession(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	x.RedirectToPublicRoute(h.r)(w, r, ps)
```

Comment:
I don't think this will work for your use case. You still need the session cookie if you do it like this, which you don't have in the backoffice, right? See https://github.com/ory/kratos/pull/2867/files#diff-abfcc5544632668aab2f1162de82ec56e8ec7444f6ce377cb3a30a9eafbed198L585

IIUC, you will need to write the logic from https://github.com/ory/kratos/pull/2867/files#diff-abfcc5544632668aab2f1162de82ec56e8ec7444f6ce377cb3a30a9eafbed198L602 here and fetch `SessionID` from this `r`s body.

Reply:
Yes, I don't think so either 😅. I just threw in that copy-pasta to finish the sdk gen

Reply:
Ah, just wanted to make sure you don't waste the time while integrating this into the backoffice 😅